wwwroot/src/NotificationClients/GraphNotificationClient.ts

```diff
@@ -118,17 +118,22 @@ export class GraphNotificationClient implements INotificationClient {
         });
 
         // message received
-        this.connection.on(this.ReturnMethods.NewMessage, async (notification) => {
-
-            // decrypt the content
-            const response = await fetch(ConfigUtil.GnbEndpoint + "/api/GetChatMessageFromNotification", {
-                method: "POST",
-                headers: {
-                    Authorization: `Bearer ${gnbToken.accessToken}`
-                },
-                body: JSON.stringify(notification.encryptedContent)
-            });
-            var decryptedResourceData:any = await response.json();
+        this.connection.on(this.ReturnMethods.NewMessage, async (notification, decryptedContent) => {
+
+            // make descryption backwards compatible
+            var decryptedResourceData:any = decryptedContent;
```

Comment:
decryptedContent is returned as string and not as JSON object. This will cause "if (decryptedResourceData.from)" in line 164 to return false and hence UI will not be updated.

Have created a PR to update this branch https://github.com/microsoft/EmbeddedTeamsChat/pull/5